bin/barclamp_lib.rb

```diff
@@ -131,7 +84,7 @@ def print_commands(cmds, spacer = "  ")
   end
 end
 
-def usage(rc)
+def usage (rc)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)